js/geo/coordinate.js

```diff
@@ -15,6 +15,7 @@ function Coordinate(column, row, zoom) {
     this.column = column;
     this.row = row;
     this.zoom = zoom;
+    this.posMatrix = null;
```

Comment:
I'm not seeing this as caching. It calculates the matrix exactly once for every tile every frame. The question is where the calculated matrix should live and how it should be accessed.

The newly implemented approach is storing it on `coord` and accessing it as `coord.posMatrix`.

An alternative would be have a `painter.calculatePosMatrices(coords)` and `painter.getPosMatrix(coord)`. It would calculate the matrices and index them by `coord.id`. This would be similar to how clipping id's are used.

The first approach is more similar to -native. I think in -native it's preferable to just access a member of the tile than having a map lookup.